tests/models/test_llama_cpp.py

```diff
@@ -3,36 +3,55 @@
 from guidance import select, gen
 from ..utils import get_model
 
-def test_llama_cpp_gen():
-    lm = get_model("llama_cpp:")
+import pytest
+
+
+@pytest.fixture(scope="session")
+def llamacpp_model(selected_model, selected_model_name):
+    if selected_model_name in ["hfllama7b"]:
+        return selected_model
+    else:
+        pytest.skip("Requires Llama-Cpp model")
+
+
+def get_llama_with_batchsize(model_info, n_batch: int = 248):
+    # Now load the model we actually want
+    my_kwargs = model_info["kwargs"]
+    my_kwargs["n_batch"] = n_batch
+    lm = get_model(model_info["name"], **my_kwargs)
+    return lm
```

Comment:
If I remember right, we don't support executing with n_batch > 1 on guidance local inference -- why 248 here? Is that the max the CI/CD machine support? Could be misremembering though!